README.md

```diff
@@ -111,7 +111,7 @@ You will be prompted to enter an export password, the again to verify.  Remember
 ### Install the certificate on server
 These installation instructions are specific to Windows 7.  If the OSIsoft server is running on a different OS the process may differ.
 
-* In Windows, right click on the certificate.crt file and click Install Certificate.
+* In Windows, right click on the certificate.crt file and click "Install Certificate" to the current user.
```

Comment:
Please merge/squash this commit with the original commit that added this line. No need to modify this line in a separate commit when the original commit that added it is in this PR.